webapp/src/jaas/components/conference/index.ts

```diff
@@ -0,0 +1,4 @@
+import {connect} from 'react-redux';
+import {JaaSConference, MapDispatchToProps} from './jaas_conference';
```

Comment:
Move the code of `jaas_conference.tsx` inside this file and convert that to a functional component so that there is only one file - `index.tsx`.
If it's not possible to convert that into functional component, then move the `MapDispatchToProps` function to this file.